scripts/sync-flow-tests.js

```diff
@@ -66,7 +65,7 @@ function syncTests(syncDir) {
     const specFile = path.join(dirname, SPEC_FILE_NAME);
     const specContent = specContents[specFile] || DEFAULT_SPEC_CONTENT;
 
-    mkdirp.sync(dirname);
+    fs.mkdirSync(dirname, { recursive: true });
```

Comment:
@fisker Thanks. Since Node.js v10 provides `fs.mkdirSync` with `recursive` option, I think `mkdirp` package is unnecessary. This part has to be changed from `mkdirp` to `fs.mkdirSync` to remove `mkdirp` package from dependencies.

Reply:
I mean just use `fs.mkdirSync` don't care `engine`.

Reply:
you mean `mkdirp` doesn't have to be removed?